src/Knp/Component/Pager/Pagination/AbstractPagination.php

```diff
@@ -2,7 +2,9 @@
 
 namespace Knp\Component\Pager\Pagination;
 
-abstract class AbstractPagination implements PaginationInterface
+use Countable, Iterator, ArrayAccess;
+
+abstract class AbstractPagination implements PaginationInterface, Countable, Iterator, ArrayAccess
```

Comment:
This one should not be updated I think?

Reply:
I don't know you tell me. It's this commit https://github.com/KnpLabs/knp-components/commit/57a3d99be8033265f22de55dc2c9525eb113dd40#diff-edbb8b71ea2ac5be6c02fb9fd282ff38 which parts from it should be applied?

Reply:
It's a BC break but it's OK because it's a next major version, so let's keep the whole commit.

Reply:
Ok, so is this good to merge?

Reply:
Yea thanks @ostrolucky :)